### PR TITLE
Mainnet deploys

### PR DESCRIPTION
**Pull request by sanderpick**
- Adds back optimism-kovan
- Deploys to ethereum, optimism, polygon (all contracts verified)
- Upgrades ethereum-goerli, optimism-kovan, polygon-mumbai, optimism-kovan-staging (verified new impl contracts sand optimism-kovan, see note below)

Note: For some reason, I'm no longer able to verify contracts on optimism kovan. Getting "invalid API key" error despite it being correct and unchanged. Will look a bit more.

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -45,14 +45,20 @@ const config: HardhatUserConfig = {
...
-      arbitrumOne: process.env.ARBISCAN_API_KEY || "",
-      arbitrumGoerli: process.env.ARBISCAN_API_KEY || "",
-
       // polygon
       polygon: process.env.POLYSCAN_API_KEY || "",
       polygonMumbai: process.env.POLYSCAN_API_KEY || "",
     },
+    customChains: [
+      {
+        network: "arbitrumGoerli",
```
Comment (joewagner) [inline hardhat.config.ts:54]: Reading through the docs here: https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan#adding-support-for-other-networks it makes it sound like we might need an entry for `etherscan.apiKey.arbitrumGoerli`?

**Review by joewagner — APPROVED** [review]
This all looks good!  I added one question in regard to the etherscan plugin and verifying on arbitrumGoerli, but we can merge either way.

File: hardhat.config.ts (hunk 1 above)
Comment (sanderpick) [reply hardhat.config.ts:57]: oh yeah, good catch. so far as i can tell, there's no way to programmatically verify the contract on arbitrum goerli yet... i think they'll have to add it to the main etherscan explorer vs. just blockscout. i added the entry though

Comment (sanderpick) [reply hardhat.config.ts:57]: looks like a need a re-approval

**Review by joewagner — APPROVED** [review]
LGTM!

**Pull request merged**